TGC/localisation/00_events_1000000.csv

```diff
@@ -659,6 +659,54 @@ EVTDESC8870779;After our brief bout of instability, the local authorities in $PR
+EVTNAME8870789;Walker asks for annexation;;;;;;;;;;;;;x
+EVTDESC8870789;William Walker has sent us a moving dispatch citing the American traditions of liberty, freedom, and buisness on why Central America should be annexed into the United States. This argument was slightly mitigated by his repeated mention of slavery and how it is necessisary for the survival of the colored race. Annexing this region would no doubt infuriate the North. Alternatively, we could allow them to keep their domestic independence while controling their foreign policy. Finally we could just decide to let them forge their own destiny. ;;;;;;;;;;;;;x
+EVTOPTA8870789;More states for the Union!;;;;;;;;;;;;;x
+EVTOPTB8870789;They will be a useful ally in the region;;;;;;;;;;;;;x
+EVTOPTC8870789;It is not worth the trouble;;;;;;;;;;;;;x
+EVTNAME8870790;Walker asks for American guidance ;;;;;;;;;;;;;x
+EVTDESC8870790;William Walker has sent us a dispatch citing the American tradition of liberty in why the United States should help guide his new state in Central America. This argument was slightly mitigated by his repeated mention of slavery and how it is necessisary for the survival of the colored race. Doing so would no doubt anger northerners who are sensitive to any issue involving slavery. Alternatively, we could just decide to let them forge their own destiny. ;;;;;;;;;;;;;x
+EVTOPTA8870790;They will be a useful ally in the region;;;;;;;;;;;;;x
+EVTOPTB8870790;It is not worth the trouble;;;;;;;;;;;;;x
+EVTNAME8870791;Walker asks for American guidance ;;;;;;;;;;;;;x
+EVTDESC8870791;William Walker has sent us a curt dispatch wishing for cooperative and trade between the United States and his Central American state. This surprised the cabinet considering Walker had discussed annexation to the United States with the media in the past. ;;;;;;;;;;;;;x
+EVTOPTA8870791;Oh well, at least they won't be controled by the British ;;;;;;;;;;;;;x
+EVTNAME8870791;Walker wishes to join the Confederacy!;;;;;;;;;;;;;x
+EVTDESC8870791;Walker has sent us a moving dispatch citing the American traditions of liberty, freedom, and buisness on why Central America should join the Confederacy. He emphasized his pro-slavery attitudes with discussions on how slavery is necessisary for the survival of the colored race. We could annex this region directly, expanding our nation. Alternatively we could allow it to keep its indepdences in order to gain access to its already hardened military tradition. ;;;;;;;;;;;;;x
+EVTOPTA8870791; Manifest Destiny will extend southward! ;;;;;;;;;;;;;x
+EVTOPTB8870791; It helps us more to allow local administration ;;;;;;;;;;;;;x
+EVTOPTC8870791; We do not wish to associate with Spanish speakers ;;;;;;;;;;;;;x
+EVTDESC8916799;$FROMCOUNTRY$ has agreed to sell us all lands east of the St. Lawrence river. The borders of New England continue to grow, and business continue to boom. Another victory for Brother Jonathan in Boston and his deep pockets!;;;;;;;;;;;;;x
```

Comment:
You already have this (EVTDESC8916799) line above. Copy paste accident maybe. Also you have the NAME and OPTA of that same event below, breaking id ordering.